buildSrc/src/main/resources/gocd-docker-agent/README.md.ftl

```diff
@@ -100,6 +100,9 @@ In this case, as the docker deamon will be the one mounting the volumes you defi
 If you run several agents container, you will need to overwrite the `VOLUME_DIR` environment variable to have a different path for your `/godata` for each of your gocd agent containers (to avoid issues). For example, if the volume on your host for the first container is `/go-agent1/godata`, you will set the `VOLUME_DIR` environment data on your container to `/go-agent1/godata` and the `docker-entrypoint.sh` script will automatically manage it and make sure the agent stores its configuration, logs and pipelines there.
 </#if>
 
+# Running GoCD Containers as Non Root
```

Comment:
Always remember to add a blank line between sections in markdown. Makes it a huge problem when different parsers parse this markdown and interpret it differently.

Reply:
Thanks, will keep this in mind :)